### PR TITLE
Fix problem with reviews on future presentations

### LINKED ISSUE #104: Sending /lab1feedback for classes not yet finished
If we indicate `datetime` in `/lab1feedback [datetime]` outside the suggested, it will provide us with presentations that were scheduled on that day.

Screenshot 2023-12-06 at 14 53 12

Screenshot 2023-12-06 at 14 50 49

Then we can rate them.
Screenshot 2023-12-06 at 15 00 14

### PR DESCRIPTION
**Pull request by Ivanio1**
Close #104

### CODE REVIEW

File: src/course_bot/presentation.clj [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-             [course-bot.talk :as talk]))
+(ns course-bot.presentation
+  (:require [clojure.java.io :as io]
+            [clojure.pprint :as pprint]
+            [clojure.string :as str])
+  (:require [codax.core :as codax])
+  (:require [course-bot.general :as general]
+            [course-bot.internationalization :as i18n :refer [tr]]
+            [course-bot.misc :as misc]
+            [course-bot.talk :as talk]))
```
Comment (Ivanio1) [inline src/course_bot/presentation.clj:10]: It was fixed by cljfmt.exe

File: src/course_bot/presentation.clj [hunk 2]
```diff
@@ -602,6 +604,10 @@
             (talk/send-text token id (tr :pres/already-received))
             (talk/stop-talk tx))
 
+          (when (> (misc/read-time dt) now)
+            (talk/send-text token id (tr :pres/too-early))
```
Comment (ryukzak) [inline src/course_bot/presentation.clj:608]: Do you mean" "You can't five a feedback to the future lesson"?

Also, we need a test for it.

**Review by ryukzak — CHANGES_REQUESTED** [review]

**Comment by Ivanio1** [comment]
All fixed.

**Comment by ryukzak** [comment]
Sorry, but I can't bring you more scores for the quests.

**Pull request merged**